src/runtime-rs/crates/resource/src/volume/mod.rs

```diff
@@ -41,10 +45,15 @@ impl VolumeResource {
         &self,
         share_fs: &Option<Arc<dyn ShareFs>>,
         cid: &str,
-        oci_mounts: &[oci::Mount],
+        spec: &oci::Spec,
     ) -> Result<Vec<Arc<dyn Volume>>> {
         let mut volumes: Vec<Arc<dyn Volume>> = vec![];
+        let oci_mounts = &spec.mounts;
+        // handle huge page options
+        let options_map = get_huge_page_option(spec)?;
```

Comment:
1. Suggest to add `context()` for `get_huge_page_option()`.
2. Suggest calling `get_huge_page_option()` in is_huge_page section after line 69.

Reply:
fixed